pages/common/man.md

```diff
@@ -6,6 +6,10 @@
 
 `man {{command}}`
 
+- Display man page for a command from section 7:
+
+`man {{command}}.7`
```

Comment:
Perhaps we could put `7` in `{{` and `}}`?

Reply:
oh yeh that makes sense

Reply:
In several examples, I see `man command <section>` being used. Is that incorrect ?

Reply:
On Linux, `man command <section>` will pull up two man pages: one for command, and one for section(which will undoubtedly not exist).

Reply:
Ah thanks !